src/util.js

```diff
@@ -155,98 +168,100 @@ class Logger {
         if (this.hideLog[level] && this.hideLog[level].includes(module.toLowerCase())) {
             return;
         }
-        module = module.toUpperCase();
-        level = level.toUpperCase();
         let now;
         if (dayjs_1.default.tz) {
             now = dayjs_1.default.tz(new Date()).format();
         }
         else {
             now = (0, dayjs_1.default)().format();
         }
+        const structuredLogging = process.env.UPTIME_KUMA_LOG_FORMAT === "json";
         const levelColor = consoleLevelColors[level];
         const moduleColor = consoleModuleColors[intHash(module, consoleModuleColors.length)];
-        let timePart;
-        let modulePart;
-        let levelPart;
-        let msgPart;
-        if (exports.isNode) {
-            switch (level) {
-                case "DEBUG":
-                    timePart = exports.CONSOLE_STYLE_FgGray + now + exports.CONSOLE_STYLE_Reset;
-                    break;
-                default:
-                    timePart = exports.CONSOLE_STYLE_FgCyan + now + exports.CONSOLE_STYLE_Reset;
-                    break;
+        let timePart = now;
+        let modulePart = module;
+        let levelPart = level;
+        let msgPart = msg;
+        if (!structuredLogging) {
+            module = module.toUpperCase();
+            if (exports.isNode) {
+                switch (level) {
+                    case "DEBUG":
+                        timePart = exports.CONSOLE_STYLE_FgGray + now + exports.CONSOLE_STYLE_Reset;
+                        break;
+                    default:
+                        timePart = exports.CONSOLE_STYLE_FgCyan + now + exports.CONSOLE_STYLE_Reset;
+                        break;
+                }
+                modulePart = "[" + moduleColor + module + exports.CONSOLE_STYLE_Reset + "]";
+                levelPart = levelColor + `${level}:` + exports.CONSOLE_STYLE_Reset;
+                switch (level) {
+                    case "ERROR":
+                        if (typeof msg === "string") {
+                            msgPart = exports.CONSOLE_STYLE_FgRed + msg + exports.CONSOLE_STYLE_Reset;
+                        }
+                        break;
+                    case "DEBUG":
+                        if (typeof msg === "string") {
+                            msgPart = exports.CONSOLE_STYLE_FgGray + msg + exports.CONSOLE_STYLE_Reset;
+                        }
+                        break;
+                    default:
+                        break;
+                }
+            }
+            else {
+                timePart = now;
+                modulePart = `[${module}]`;
+                levelPart = `${level}:`;
```

Comment:
This is alredy the value, right?

```suggestion
                modulePart = `[${module}]`;
```

Reply:
You're right, the `timePart` is redundant. The other two are actually modifying things though.

Reply:
Good Point, overlooked the second one